### DISCUSSION TITLE
Extensions for Ebola vignette

### DISCUSSION THREAD

**Discussion by athowes** [epinowcast/epidist#550] [Announcements]
The Ebola vignette (#52) is really big. So for now we will cut the parts about Gamma and model comparison. I'm going to paste things here that are cut, and eventually write out possible extensions.

Edit:

So my read of the possible extension is:

* Model comparison with `loo` to show that (probably) sex-district stratified model is best
* Fit gamma delay with best model
* Again model comparison for gamma vs. lognormal

**Comment by athowes** [comment]
```md
2. Fitting models with lognormal and gamma delay distributions.
4. Selecting between fitted models using the [`loo`](https://mc-stan.org/loo/) package.
```

```md
## Model comparison

* Use `loo` and PSIS
* Find out if it's worth including sex as a covariate. Presumably yes given how different the delay distributions are
* Find out if it's worth including district as a random effects
  * What about spatially structured? Probably future work: how to include more structured random effects

# Using a gamma delay distribution

## Fitting 

* Just fit the best model from above here? Yes
* Going to do the sex stratified one for now

fit_sex_gamma <- epidist(
  data = obs_prep,
  formula = brms::bf(mu ~ 1 + sex, shape ~ 1 + sex),
  family = stats::Gamma(),
  algorithm = "laplace",
  refresh = 0,
  silent = 2,
  seed = 1
)

summary(fit_sex_gamma)

## Model selection

* Should we use the lognormal or gamma delay distribution?
```

```md
* The likelihood family may also be specified in `epidist`
* Model comparison may be performed via the `loo` package
* Factors that we did not take into account in this vignette include
  * Age: this would be a simple extension of the model
  * Time: it is less clear how appropriate it is to include time as a covariate in the model. That said, I think it is an important future direction because I expect the time point during the epidemic to be a clear confounder of delay duration. Given epidemic start time varies by district, the "district" effects in the model currently are complicated by this fact
```

**Comment by athowes** [comment]
Adding issues here left from clean-up:

* [ ] https://github.com/epinowcast/epidist/pull/52#discussion_r1742068658 Should we have facet specific x-axis for the plot showing the data. I prefer no, Katie prefers yes.
* [ ] https://github.com/epinowcast/epidist/pull/52#discussion_r1742271807 Could we simplify ptime and ptime_daily. See
https://github.com/epinowcast/epidist/issues/291 also
* [ ] https://github.com/epinowcast/epidist/pull/52#discussion_r1743966633 Better explainations of what is happening in the PMF and PDF prediction generation. From my perspective, I understand them as being predictions of something. Why this is exactly what you want for policy purposes I understand less. For example, why do you want to predict what you would observe with censoring for the PMF, rather than what you would observe without censoring and then discretise that? I somewhat get it but not all the way (and if I'm confused then readers will be also I think). This relates to the "going around in circles"-esque discussion on the initial vignette and follow-up issue e.g. https://github.com/epinowcast/epidist/issues/67

**Comment by athowes** [comment]
Note @zsusswein expressed interest F2F in there being model comparison in this vignette!

**Comment by seabbs** [comment]
I also heard about this and heard posterior predictions vs real data which makes a lot of sense

**Comment by zsusswein** [comment]
As a reader, I would also enjoy some nice prior predictive visualizations

**Comment by seabbs** [comment]
There is a balance to be struck here with complexity vs useful info so may need to watch that and in some cases just signpost out to other vignettes where these details are covered.

**Comment by seabbs** [comment]
Moving this to a discussion
